scripts/verify-install.sh

```diff
@@ -9,17 +9,23 @@ MANIFEST_SELECTOR=". | select(.name | test(\"manifest-v.*(\\\\.txt)$\")) | .name
 SIGNATURE_SELECTOR=". | select(.name | test(\"manifest-.*(\\\\.sig)$\")) | .name"
 HEADER_JSON="Accept: application/json"
 HEADER_GH_JSON="Accept: application/vnd.github.v3+json"
+MIN_REQUIRED_SIGNATURES=5
 
-# All keys that can sign lnd releases. The key must be downloadable/importable
-# from the URL given after the space.
+# All keys that can sign lnd releases. The key must be added as a file to the
+# keys directory, for example: scripts/keys/<username>.asc
+# The username in the key file must match the username used for signing a
+# manifest (manifest-<username>-v0.xx.yy-beta.sig), otherwise the signature
+# won't be counted.
 KEYS=()
-KEYS+=("F4FC70F07310028424EFC20A8E4256593F177720 https://keybase.io/guggero/pgp_keys.asc")
-KEYS+=("15E7ECF257098A4EF91655EB4CA7FE54A6213C91 https://keybase.io/carlakirkcohen/pgp_keys.asc")
-KEYS+=("9C8D61868A7C492003B2744EE7D737B67FA592C7 https://keybase.io/bitconner/pgp_keys.asc")
-KEYS+=("E4D85299674B2D31FAA1892E372CBD7633C61696 https://keybase.io/roasbeef/pgp_keys.asc")
-KEYS+=("729E9D9D92C75A5FBFEEE057B5DD717BEF7CA5B1 https://keybase.io/wpaulino/pgp_keys.asc")
-KEYS+=("7E81EF6B9989A9CC93884803118759E83439A9B1 https://keybase.io/eugene_/pgp_keys.asc")
-KEYS+=("7AB3D7F5911708842796513415BAADA29DA20D26 https://keybase.io/halseth/pgp_keys.asc")
+KEYS+=("F4FC70F07310028424EFC20A8E4256593F177720 keys/guggero.asc")
+KEYS+=("15E7ECF257098A4EF91655EB4CA7FE54A6213C91 keys/carlakirkcohen.asc")
+KEYS+=("9C8D61868A7C492003B2744EE7D737B67FA592C7 keys/bitconner.asc")
+KEYS+=("E4D85299674B2D31FAA1892E372CBD7633C61696 keys/roasbeef.asc")
+KEYS+=("729E9D9D92C75A5FBFEEE057B5DD717BEF7CA5B1 keys/wpaulino.asc")
+KEYS+=("7E81EF6B9989A9CC93884803118759E83439A9B1 keys/eugene_.asc")
+KEYS+=("7AB3D7F5911708842796513415BAADA29DA20D26 keys/halseth.asc")
```

Comment:
Nit: single source of truth recommends stripping away `key/` prefix and `.asc` suffix. Apart from avoiding typos it happens to make the code below simpler.